packages/compass-home/src/components/home.tsx

```diff
@@ -0,0 +1,209 @@
+/** @jsx jsx */
```

Comment:
Spent some time trying to get the `css` prop working with babel and couldn't get it to the place where tsconfig, ts lint, and the webpack config were working well together. I think I'd like to address this in a follow up or separate PR if that's cool.

https://emotion.sh/docs/css-prop
https://emotion.sh/docs/typescript#css-prop

Using `"jsxImportSource": "@emotion/react"` in our `compilerOptions` in the `tsconfig` was erroring. I think it might be something that needs react 17 to work nicely without some more intentional configuration https://www.typescriptlang.org/tsconfig#jsxImportSource